network/load-balancer/concepts.mdx

```diff
@@ -111,23 +137,23 @@ Proxy protocol is an internet protocol used to transfer connection information f
 
 ## SSL bridging
 
-[SSL bridging](/network/load-balancer/how-to/setup-ssl-offloading#how-to-#how-to-configure-tlsssl-bridging) removes SSL-based encryption from HTTPS traffic coming into the Load Balancer (as with SSL offloading), but then initiates a new SSL connection to rencrypt traffic between the Load Balancer and the backend servers.
+[SSL bridging](/network/load-balancer/reference-content/configuring-load-balancer#configuring-a-load-balancer-for-ssl-bridging) removes SSL-based encryption from HTTPS traffic coming into the Load Balancer (as with SSL offloading), but then initiates a new SSL connection to re-encrypt traffic between the Load Balancer and the backend servers.
 
 </Concept>
 
 <Concept>
 
 ## SSL offloading
 
-[SSL offloading](/network/load-balancer/how-to/setup-ssl-offloading#how-to-configure-tlsssl-offloading) removes SSL-based encryption from HTTPS traffic coming into the Load Balancer.
+[SSL offloading](/network/load-balancer/reference-content/configuring-load-balancer#configuring-a-load-balancer-for-ssl-offloading) removes SSL-based encryption from HTTPS traffic coming into the Load Balancer.
```

Comment:
Again here it gives the feeling we are just talking about SSL which is not the case. SSL offloading is a pattern where the load balancer terminates encrypted connection at the frontend, effectively decrypting incoming traffic with the intention to forward it unencrypted to the backend servers. This effectively offloads the work of decrypting traffic from the backend server to the load balancer.